stanford-earth/stanford_news
# EARTH-932: Added imagesloaded to masonry reshuffle on earth matters

# READY FOR REVIEW

# Summary
- Re-shuffles the layout after each image is loaded to ensure no overlaps.

# Needed By (Date)
- ?

# Urgency
- Medium?

# Steps to Test

1. Check out this branch
2. Clear all caches
3. Navigate to the earth matters page
4. Enable chrome dev tools (`cmd + i`)
5. Turn on the mobile device tool (`cmd + shift + m`)
6. Set the throttle to `mid-tier mobile` or low. This is found at the top of the page under the `online` dropdown
7. Refresh the page and scroll down watching the items load and sort
8. Click on the load more button and resize the screen while watching them load to a table size.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
